README.md

```diff
@@ -5,70 +5,102 @@ author: jmprieur
-6. While still in the Azure portal, choose your application, click on **Settings** and choose **Properties**.
-7. Find the Application ID value and copy it to the clipboard.
-8. In the same page, change the `logoutUrl` property to `https://localhost:44353/Account/EndSession`.  This is the default single sign out URL for this sample.
-9. For the App ID URI, enter `https://<your_tenant_name>/WebApp-OpenIDConnect-DotNet`, replacing `<your_tenant_name>` with the name of your Azure AD tenant. 
+- To create the sample from the command line, execute the following command:
+
+  ```console
+  dotnet new mvc --auth SingleOrg --client-id <CLIENT_ID_(APP_ID)> --tenant-id <TENANT_ID> --domain <TENANT_DOMAIN>
+  ```
+  Use the values that you recorded from the Azure portal for \<CLIENT\_ID\_(APP\_ID)>, \<TENANT\_ID>, and \<TENANT\_DOMAIN>.
+
+- To clone/download this sample, execute the following command from your shell or command line:
+
+  ```console
+  git clone https://github.com/Azure-Samples/active-directory-dotnet-webapp-openidconnect-aspnetcore.git
+  ```
 
-### Step 3:  Configure the sample to use your Azure Active Directory tenant
+  In the **appsettings.json* file, provide values for the `Domain`, `TenantId`, and `ClientID` that you recorded earlier from the Azure portal.
 
-1. Open the solution in Visual Studio 2017.
-2. Open the `config.json` file.
-3. Find the `Tenant` property and replace the value with your AAD tenant name.
-4. Find the `ClientId` and replace the value with the Application ID from the Azure portal.
-5. If you changed the base URL of the sample, find the app key `ida:PostLogoutRedirectUri` and replace the value with the new base URL of the sample.
+If you aren't using [Visual Studio](https://www.visualstudio.com/vs/) as your development environment and created the project with the `dotnet new mvc` command:
 
-### Step 4:  Run the sample
+- Add Bower to the project file:
+  ```xml
+  <Target Name="PreBuildScript" BeforeTargets="PrepareForBuild">
+    <Exec Command="bower install" />
+  </Target>
+  ```
+- Add `BundlerMinifer.Core` to the `ItemGroup` containing `DotNetCliToolReference` references:
+  ```xml
+  <DotNetCliToolReference Include="BundlerMinifier.Core" Version="2.5.357" />
+  ```
+- Add the `dotnet bundle` command target to bundle and minify scripts and styles in the app:
+  ```xml
+  <Target Name="PrePublishScript" BeforeTargets="PrepareForPublish">
+    <Exec Command="dotnet bundle" />
+  </Target>
+  ```
 
-Clean the solution, rebuild the solution, and run it.
+### Step 3: Run the sample
 
-Click the sign-in link on the homepage of the application to sign-in.  On the Azure AD sign-in page, enter the name and password of a user account that is in your Azure AD tenant.
+Build the solution and run it.
 
-## About The Code
+Make a request to the app. The app immediately attempts to authenticate you via Azure AD. Sign in with your Global Administrator account.
```

Comment:
Why do we need to be a global admin account? that should work with any account of the tenant?